src/Admin/AbstractAdmin.php

```diff
@@ -2297,7 +2297,7 @@ final protected function appendParentObject(object $object): void
 
         if ($this->hasParentFieldDescription()) {
             $parentAdmin = $this->getParentFieldDescription()->getAdmin();
-            $parentObject = $parentAdmin->getObject($this->getRequest()->get($parentAdmin->getIdParameter()));
+            $parentObject = $this->hasRequest() ? $parentAdmin->getObject($this->getRequest()->get($parentAdmin->getIdParameter())) : null;
```

Comment:
You have the same logic before with
```
if ($this->isChild()) {
            $parentAssociationMapping = $this->getParentAssociationMapping();
            if (null !== $parentAssociationMapping) {
                $parentAdmin = $this->getParent();
                $parentObject = $parentAdmin->getObject($this->getRequest()->get($parentAdmin->getIdParameter()));
```


If we add a hasRequest check we might need to add one before no ?


Also it could be simpler with

```
if ($this->hasParentFieldDescription() && $this->hasRequest()) {
```